### PR TITLE
Fix typo errors

### PR DESCRIPTION
**Pull request by cedricdelpoux**

### CODE REVIEW

**Comment by jxnblk** [comment]
Thanks, checking this out now!

File: docs/api.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Uses [Palx](https://github.com/jxnblk/palx) to generate a full UI color palette.
 - `borderLeft` - (string, number, or boolean) same as the `border` prop but for border-left
 - `borderColor` - (string) sets border color based on a color scheme key or raw value
 
-### Border Radii
```
Comment (jxnblk) [inline docs/api.md]: This one isn't a typo. Why did you change this?

Comment (cedricdelpoux) [reply docs/api.md]: Oh sorry I was thinking it was. I will revert this

**Comment by cedricdelpoux** [comment]
It should be good

**Comment by cedricdelpoux** [comment]
waiting for you

**Comment by cedricdelpoux** [comment]
Why did you open source you project if you do not accept help ? 👏🏻
Thank you, we abandoned to use your project because of your reactivity
